package.json

```diff
@@ -103,7 +103,6 @@
     "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION"
   },
   "devDependencies": {
-    "@asyncapi/react-component": "^2.0.0",
```

Comment:
I guess you have some special setup to remove this.. bc it's not going to work otherwise

Reply:
yes, i am waiting for my pr in `@asyncapi/react-component` to get merged. Then I will update the version according to it and push it. This shouldnt be merged untill then.

Reply:
@ivangsa ptal.